tools/tune/dmic/dmic_fir_export_alsa.m

```diff
@@ -0,0 +1,104 @@
+% dmic_fir_export - Export FIR coefficients
+%
+% [success, vfn_arr] = pdm_export_coef(fir, hdir, prev_vfn_arr)
+%
+% fir     - fir definition struct
+% hdir    - directory for header files
+
+% SPDX-License-Identifier: BSD-3-Clause
+%
+% Copyright (c) 2023, Intel Corporation. All rights reserved.
+%
+% Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
+
+function [success, vfn_arr] = dmic_fir_export_alsa(fir, hdir, prev_vfn_arr)
+
+if ~exist(hdir, 'dir')
+	mkdir(hdir);
+end
+
+pbi = round(fir.cp * 1e4);
+sbi = round(fir.cs * 1e4);
+rpi = round(fir.rp * 1e2);
+rsi = round(fir.rs);
+
+hfn = sprintf('%s/pdm-decim-fir.h', hdir);
+vfn = sprintf('fir_int32_%02d_%04d_%04d_%03d_%03d', fir.m, pbi, sbi, rpi, rsi);
+sfn = sprintf('pdm_decim_int32_%02d_%04d_%04d_%03d_%03d', fir.m, pbi, sbi, rpi, rsi);
+
+fprintf('Exporting %s ...\n', hfn);
+
+if exist(hfn) == 2
```

Comment:
I believe there is a second argument missing!

Reply:
yep, useful here too.